### ISSUE TITLE
getting a warning unique key?

### ISSUE THREAD

**Issue by chiefjester** [clayallsopp/relay-101#1]
`Warning: Each child in an array or iterator should have a unique "key" prop. Check the render method of`TopItems`. See https://fb.me/react-warning-keys for more information.`

I was reading the medium post on this at first maybe I mistyped or something.  Then I cloned this repo.  I'm getting the same error about unique key prop. 

Error disappers if I remove `{ item }` which is basically just calling Item Component

```
class Item extends React.Component {
  render() {
    let item = this.props.store;
    return (
      <div key={item.id}>
        <h1><a href={item.url}>{item.title}</a></h1>
        <h2>{item.score} - {item.by.id}</h2>
        <hr />
      </div>
    );
  }
};
```

Any idea?

**Comment by clayallsopp** [comment]
sorry about that! fixed by #2 

I also updated the medium post accordingly

**Comment by chiefjester** [comment]
thanks and :+1:
